### PR TITLE
Added Tenant ID Passthrough

### PR DESCRIPTION
**Pull request by gaokevin1**
## Related Issues

✅ Fixes https://github.com/descope/etc/issues/3948

## Description

This will allow you to pass in a Tenant ID, and use SSO associated with that tenant, without having to pass in an email with a screen in the flow.

## Screenshots

📺🔫 All of the UI influenced by this PR

## Must

- [ ] 📱 Responsiveness (mobile/XL resolutions)
- [X] 🧪 Tests
- [X] 📃 Documentation (if applicable)

### CODE REVIEW

**Review by omercnet — DISMISSED** [review]

File: .env.example [hunk 1]
```diff
@@ -3,3 +3,4 @@ DESCOPE_FLOW_ID="sign-up-or-in" # Your Descope flow ID
 DESCOPE_FLOW_DEBUG= # Set to true in case you want to debug your flow
 REACT_APP_DESCOPE_BASE_URL= # Descope API base URL
 REACT_APP_USE_ORIGIN_BASE_URL= # Set in case you want to use the origin as
+DESCOPE_TENANT_ID= # Descope tenant ID
```
Comment (talaharoni) [inline .env.example:6]: I suggest to have it after the `DESCOPE_FLOW_ID`

File: README.md [hunk 2]
```diff
@@ -42,6 +43,7 @@ You can refer to either the [main documentation](https://docs.descope.com/custom
 `<PROJECT_ID>` as part of the URL path is required to use the desired Descope's `PROJECT_ID`  
 `flow` query parameter is optional. If none provided the default flow is `sign-up-or-in`  
 `debug` query parameter is optional. If debug mode is needed use `debug=true`
+`tenant` query parameter is optional. If present, then you will be able to authenticate via SSO, without having to first specify an email with an input screen in your flow.
```
Comment (talaharoni) [inline README.md:46]: same, I suggest to have it after the `flow`

Comment (talaharoni) [reply README.md:46]: It should work with either tenantID/tenantName/emailDomain right?
Did you test it? maybe worth adding it to the README.

Comment (gaokevin1) [reply README.md]: Changed the README. Tenant Name didn't seem to work but Tenant ID and email domain works, so I added to the README.

File: .env.example (hunk 1 above)
Comment (gaokevin1) [reply .env.example]: Done

**Review by omercnet — APPROVED** [review]

**Pull request merged**